### DISCUSSION TITLE
Build: an error has occurred, exit

### DISCUSSION THREAD

**Discussion by jbswdl** [LSPosed/MagiskOnWSALocal#247] [Q&A] [answered]
CUID#10 - Redirecting to https://raw.githubusercontent.com/topjohnwu/magisk-files/25.2/app-release.apk
[#e930ec 0B/0B CN:1 DL:0B]
11/10 02:36:56 [NOTICE]
![image](https://user-images.githubusercontent.com/52515996/200913922-497093f2-1a1f-4a2d-a015-de130f236f32.png)

**Comment by Howard20181** [comment, marked as answer]
Check your Internet and try again.
